docs/UserGuide.adoc

```diff
@@ -31,95 +43,127 @@ Jump to <<Quick Start>> to get started. Enjoy!
 .  Copy the file to the folder you want to use as the home folder for your Cooking Papa.
 .  Double-click the file to start the app. The GUI should appear in a few seconds.
 +
+.CookingPapa User Interface
 image::Ui.png[width="790"]
+[caption="Figure 1: ]
```

Comment:
I think we don't need to put caption for images associated to a step?

Reply:
I think if we label it as figure 1, we should explicitly label what it shows too. But I think if the picture is for a step, then there may not be a need for it